app/adapters/row.js

```diff
@@ -0,0 +1,23 @@
+import DS from 'ember-data';
+import carto from '../utils/carto';
+import generateProfileSQL from '../queries/profile';
+import decennialProfile from '../queries/decennial-profile';
+
+export default DS.JSONAPIAdapter.extend({
+  query(store, modelType, query) {
+    const { geoids, comparator, type, category } = query;
+    let selectionSQL;
+
+    if (type === 'decennial') {
+      selectionSQL = decennialProfile(geoids, category);
```

Comment:
No comparator?  Comparisons on decennial have not been implemented yet it seems.

Reply:
Not yet implemented